libindy/src/commands/mod.rs

```diff
@@ -80,9 +96,14 @@ pub fn indy_set_runtime_config(config: IndyConfig) {
     }
 }
 
+fn get_cur_time() -> u64 {
+    let since_epoch = SystemTime::now().duration_since(UNIX_EPOCH).expect("Time has gone backwards");
```

Comment:
If I understand it right this is a potential panic, which is not okay for production code

Reply:
I used this way from another production code. I can change error case to returning zero. But in any case it's a critical situation when we can't get a system time, in my opinion.